torch/_subclasses/fake_tensor.py

```diff
@@ -209,6 +209,10 @@ def maybe_get_fake_mode(t):
         m = modes[0]
         assert all(m is x for x in modes)
         return m
+    elif isinstance(t, torch.Tensor) and torch._is_functional_tensor(t):
+        reapply_views = torch._C._functionalization_reapply_views_tls()
+        unwrapped = torch._C._functorch._unwrap_functional_tensor(t, reapply_views)
+        return maybe_get_fake_mode(unwrapped)
```

Comment:
oh?! What did you need this for?

Reply:
As you can see, it merely completes `maybe_get_fake_mode` to be on par with `is_fake` when the tensor is functional.